graph_snyk_cve_sync.py

```diff
@@ -87,6 +96,10 @@ def is_dry_run(self):
         """Return True if this is a dry run."""
         return os.environ.get('DRY_RUN', 'false').lower() in ('1', 'yes', 'true')
 
+    def is_ingestion_on(self):
+        """Return if ingestion mode is on."""
+        return os.environ.get('SNYK_INGESTION_FORCE', 'false').lower() in ('1', 'yes', 'true')
```

Comment:
Environment Variable name is bit confusing.. `SNYK_INGESTION_FORCE` 
We are not Forcing ingestion. It can be more descriptive like.. simply `INGEST_SNYK_FEED`= true`

Reply:
The idea is to forcefully start the ingestion. Hence the name 'force'. Otherwise its going to anyway ingest on the stipulated time. This is to allow anyone using it on dev cluster so that they dont need to change the code.

Reply:
The idea is to over ride the scheduled time clause and forcefully run the ingestion

Reply:
We cannot add this as a job. As jobs are scheduled only via jobs API repo. This as per the definition, will continue to remain a cron

Reply:
No you are wrong. Ingestion is controlled by the time. the flag is just to over ride the timing constraint

Reply:
maybe, the name of the function can be changed